src/System.Private.ServiceModel/src/System/IdentityModel/Claims/ClaimSet.cs

```diff
@@ -48,7 +46,9 @@ public static ClaimSet Windows
                     s_windows = new DefaultClaimSet(claims);
                 }
                 return s_windows;
-#endif // FEATURE_NETNATIVE
+#else 
+                throw ExceptionHelper.PlatformNotSupported("Windows Stream Security is not supported on UWP yet");
+#endif // SUPPORTS_WINDOWSIDENTITY
```

Comment:
It's all through the code. I'll go through and update each exception message to stop at the word "supported". If only this was a string resource or a constant, it would be so much easier. What do you think about adding a constant to ExceptionHelper for this message?